sdk/analytics/src/test/java/com/klaviyo/analytics/networking/KlaviyoApiClientTest.kt

```diff
@@ -87,6 +90,17 @@ internal class KlaviyoApiClientTest : BaseRequestTest() {
         }
     }
 
+    @After
```

Comment:
Small thing, but I grouped setup and cleanup functions together for readability.